lib/field.js

```diff
@@ -42,17 +42,8 @@ export default class Field {
     this.trigger('pristine');
   }
 
-  setValue(newValue) {
-    return this.parse(newValue);
-  }
-}
-
-////////////////
-
-function constructorValues() {
-  const proxy = new Proxy(this, {
-    get: (target, property) => target[property],
-    set: (target, property, newValue) => {
+  setProxy(proxy) {
```

Comment:
This method is missing a test which tests that a fieldClass inheriting from field can overload setProxy and change its behaviour